wallet-new/src/Cardano/Wallet/Kernel/DB/HdWallet.hs

```diff
@@ -431,6 +447,14 @@ assumeHdRootExists _id = return ()
 assumeHdAccountExists :: HdAccountId -> Update' HdWallets e ()
 assumeHdAccountExists _id = return ()
 
+{-------------------------------------------------------------------------------
+  General-utility functions
+-------------------------------------------------------------------------------}
+
+-- | Computes the 'HdRootId' from the given 'EncryptedSecretKey'.
```

Comment:
We need a comment here (or a reference to a comment) why `Core.makePubKeyAddressBoot` is okay here ("held together by ducktape").